examples/qec/shors_code.py

```diff
@@ -0,0 +1,88 @@
+"""
```

Comment:
nit: minus new line - all the examples start with `""" Text of the example` without the newline

Reply:
done